Active_Record/db/migrate/20191208094805_create_sups.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSups < ActiveRecord::Migration[5.1]
+  
+  def up
+    create_table :sups do |t|
+      t.string :model
```

Comment:
It's a good practice if varchar fields have option `null: false, default: ''`

Reply:
done